src/howTos/dev/services.md

```diff
@@ -86,32 +86,34 @@ cmd: /home/alice/.cozy/scripts/konnector-node-run.sh
 url: file:///home/alice/.cozy/storage
 ```
 
-### Create the script to store the logs
+### Create the script to execute the service
 
-Create the file `~/.cozy/scripts/konnector-node-run.sh`:
-
-```bash
-#!/bin/bash
-
-rundir="${1}"
-cd $rundir
-node index.js | tee ~/.cozy/services.log
-```
+Copy the file `cozy-stack/scripts/konnector-node-run.sh` to `~/.cozy/konnector-node-run.sh`:
 
 Then you need to `chmod +x ~/.cozy/scripts/konnector-node-run.sh`
 
 Be sure to have `node` in your `/usr/bin` or `/usr/local/bin` folder. If not, you can add a symlink to `node` in one of those folder, for example by typing `ln -s $(which node) /usr/local/bin/node`
 
+### Get your service logs in a isolated file
+
+Edit your `~/.cozy/konnector-node-run.sh` by adding a tee output.
+
+```bash
+node "${arg}" | tee -a ~/.cozy/services.log
+```
+
 Now you can `tail -f ~/.cozy/services.log` to watch logs in real time.
 
-## Install your app
+## Install your app/konnector
 
 To install the app containing the service on your local stack, you must give the path of your build:
 
 ```bash
-cozy-stack apps install <app_name> file://<build_path>
+cozy-stack apps install <app_name> file:///home/toto/<rest_of_your_build_path>
```

Comment:
when I read that, it seems that `/home/toto` is mandatory. I would keep the command like it is, and add another example line, e.g. `cozy-stack apps install banks file:///home/alice/cozy-banks/build`